basic_test.sh

```
@@ -0,0 +1,74 @@
+        ((SUCCESS++))
+    fi
+done
+echo "✓ $SUCCESS/10 requests succeeded"
+
+sleep 1
+
+echo ""
+echo "Test 3: Some concurrent requests (5)"
+SUCCESS=0
+for i in {1..5}; do
+    (curl -s -o /dev/null -w "%{http_code}\n" http://localhost:8080/ 2>&1 > /tmp/curl_$i.out && echo "200" > /tmp/curl_$i.status || echo "FAIL" > /tmp/curl_$i.status) &
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)